src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -146,6 +146,8 @@ public static IServiceCollection AddApplicationInsightsTelemetry(this IServiceCo
                     excludedDomains.Add("core.chinacloudapi.cn");
                     excludedDomains.Add("core.cloudapi.de");
                     excludedDomains.Add("core.usgovcloudapi.net");
+                    excludedDomains.Add("localhost");
+                    excludedDomains.Add("127.0.0.1");
```

Comment:
Yes, this is the same change as made in Web as well. We don't want to break customers by default. Developers using other services in local machine, and remove this to monitor those calls. (not perfect either, but better than the experience of all emulator calls failing by default)